docs/nodes/nodes-library/nodes/Spotify/README.md

```diff
@@ -0,0 +1,62 @@
+---
```

Comment:
Could you please make sure that the formatting of the document aligns with the updated template?